components/support/rustlog/src/main/java/mozilla/components/support/rustlog/RustLog.kt

```diff
@@ -29,10 +32,20 @@ object RustLog {
      * (We say "almost" nothing, as calling this will hook up logging for the dynamic
      * library containing the Rust log hooking (and only that), as well as logging
      * a single message indicating that it completed initialization).
+     *
+     * @param crashReporter [CrashReporting] instance used for reporting 'error' log messages.
      */
-    fun enable() {
+    fun enable(crashReporter: CrashReporting? = null) {
         RustLogAdapter.enable { level, tagStr, msgStr ->
-            Log.log(levelToPriority(level), tagStr, null, msgStr)
+            val priority = levelToPriority(level)
+
+            crashReporter?.let {
+                if (priority == Log.Priority.ERROR) {
+                    it.submitCaughtException(RustErrorException(tagStr, msgStr))
```

Comment:
@grigoryk Is it fair to assume those log messages won't contain any PII? I'll still approve and land since this won't do anything until we actually pass in a `CrashReporting` instance.